src/behaviors/focusZone.ts

```diff
@@ -484,6 +509,23 @@ export function focusZone(container: HTMLElement, settings?: FocusZoneSettings):
         updateFocusedElement(event.target)
       }
     })
+    container.addEventListener(
+      'mousemove',
+      ({target}) => {
+        if (!(target instanceof HTMLElement)) {
```

Comment:
Are we using this to intentionally exclude `SVGElement`?

Reply:
Hmm, hadn't considered that.  The `focusableElements` array is of type `HTMLElement[]`, so we need to narrow the type in order to do `focusableElements.includes(target)`.  The check in this particular callback can use `instanceof Node`, but some of the other cannot be changed unless we want to switch `Element` or `Node` throughout this file, and in all of the `iterateFocusableElement` functions.